install/installer/pkg/components/public-api-server/configmap.go

```diff
@@ -102,3 +109,35 @@ func getStripeConfig(cfg *experimental.Config) (corev1.Volume, corev1.VolumeMoun
 
 	return volume, mount, path, true
 }
+
+func getPersonalAccessTokenSigningKey(cfg *experimental.Config) (corev1.Volume, corev1.VolumeMount, string, bool) {
+	var volume corev1.Volume
+	var mount corev1.VolumeMount
+	var path string
+
+	if cfg == nil || cfg.WebApp == nil || cfg.WebApp.PublicAPI == nil || cfg.WebApp.PublicAPI.PersonalAccessTokenSigningKeySecretName == "" {
+		return volume, mount, path, false
+	}
+
+	stripeSecret := cfg.WebApp.PublicAPI.PersonalAccessTokenSigningKeySecretName
```

Comment:
`stripeSecret` as a name doesn't make sense here. I think it's a copy/paste error from the method above.

Reply:
thanks, indeed a copy-paste